Doc/library/argparse.rst

```diff
@@ -786,8 +786,10 @@ how the command-line arguments should be handled. The supplied actions are:
     Namespace(foo=True, bar=False, baz=True)
 
 * ``'append'`` - This stores a list, and appends each argument value to the
-  list.  This is useful to allow an option to be specified multiple times.
```

Comment:
Perhaps keep "It is useful to allow an option to be specified multiple times." as the second sentence because it is more generally relevant.

Reply:
Make sense,. Thanks for your comment, daniel ;)